Jenkinsfile_CNP

```diff
@@ -0,0 +1,9 @@
+#!groovy
```

Comment:
err this already exists and is deployed on CNP in prod
https://github.com/hmcts/cmc-pdf-service/tree/cnp

Reply:
Thanks, I'll amend the PR and change target branch to cnp.